common/src/models/deployment.rs

```diff
@@ -23,6 +23,10 @@ pub struct Response {
     pub state: State,
     #[cfg_attr(feature = "openapi", schema(value_type = KnownFormat::DateTime))]
     pub last_update: DateTime<Utc>,
+    pub git_commit_id: Option<String>,
+    pub git_commit_msg: Option<String>,
+    pub git_branch: Option<String>,
+    pub git_dirty: Option<bool>,
```

Comment:
Can we add some attributes here for the `openapi` docs?

Reply:
Certainly, but I'm not sure what I'd need to add. I'm not familiar with openapi, and the other fields' attributes are just specifying their value type, which I didn't think was necessary for `Option<String>` and `Option<bool>`. Could you point me in the direction of some docs or something so I can understand what you're asking better? Or maybe I'm just missing something obvious, my apologies if so

Reply:
@iulianbarbu The `Option<String>` should probably be handled automatically due to being native, but when I checked the docs, it seems like the `KnownFormat`'s being used now might be in the wrong field? https://docs.rs/utoipa/latest/utoipa/derive.ToSchema.html#named-fields-optional-configuration-options-for-schema

Also, are the openapi docs being used by anyone?